manifests/init.pp

```diff
@@ -183,112 +178,100 @@
+  String $config                                                   = $rabbitmq::params::config,
+  Boolean $config_cluster                                          = $rabbitmq::params::config_cluster,
+  Stdlib::Absolutepath $config_path                                = $rabbitmq::params::config_path,
+  Boolean $config_ranch                                            = $rabbitmq::params::config_ranch,
+  Boolean $config_stomp                                            = $rabbitmq::params::config_stomp,
+  Boolean $config_shovel                                           = $rabbitmq::params::config_shovel,
+  Hash $config_shovel_statics                                      = $rabbitmq::params::config_shovel_statics,
+  String $default_user                                             = $rabbitmq::params::default_user,
+  String $default_pass                                             = $rabbitmq::params::default_pass,
+  Boolean $delete_guest_user                                       = $rabbitmq::params::delete_guest_user,
+  String $env_config                                               = $rabbitmq::params::env_config,
+  Stdlib::Absolutepath $env_config_path                            = $rabbitmq::params::env_config_path,
+  Optional[String] $erlang_cookie                                  = undef,
+  Optional[String] $interface                                      = undef,
+  Optional[String] $management_ip_address                          = undef,
+  Integer $management_port                                         = $rabbitmq::params::management_port,
+  Boolean $management_ssl                                          = $rabbitmq::params::management_ssl,
+  Optional[String] $management_hostname                            = undef,
+  Optional[String] $node_ip_address                                = undef,
+  Optional[Variant[Numeric, String]] $package_apt_pin              = undef,
+  String $package_ensure                                           = $rabbitmq::params::package_ensure,
+  String $package_gpg_key                                          = $rabbitmq::params::package_gpg_key,
+  String $package_name                                             = $rabbitmq::params::package_name,
+  Optional[String] $package_source                                 = undef,
+  Optional[String] $package_provider                               = undef,
+  Boolean $repos_ensure                                            = $rabbitmq::params::repos_ensure,
+  Boolean $manage_python                                           = $rabbitmq::params::manage_python,
+  String $rabbitmq_user                                            = $rabbitmq::params::rabbitmq_user,
+  String $rabbitmq_group                                           = $rabbitmq::params::rabbitmq_group,
+  String $rabbitmq_home                                            = $rabbitmq::params::rabbitmq_home,
```

Comment:
Stdlib::Unixpath ?

Reply:
I'm not sure if this module works at all on Windows, but I'll probably use Stdlib::Absolutepath to match the other data types